public/lang/hi.js

```diff
@@ -0,0 +1,146 @@
+export default {
+    pageTitle: "ब्लूमैप - {मानचित्र}",
```

Comment:
The `{map}` thing is a placeholder .. it will be replaced by the map-name that is currently selected ..
you don't want to translate that ^^

Reply:
Also, usually you don't translate the name of the product .. `BlueMap` in this case .. 
I don't know if this is special for this language, but please don't translate "BlueMap" if possible :)